recipe/meta.yaml

```diff
@@ -28,7 +33,7 @@ requirements:
     - pytorch                                # [build_platform != target_platform]
     - pytorch =*={{ torch_proc_type }}*      # [build_platform != target_platform]
```

Comment:
I'd try removing these two, because you're picking up pytorch from the build environment rather than host.

Reply:
Thanks. Trying that now. But then I am confused, does that mean you don't need arm64 pytorch at all for the cross compilation?

Reply:
Do you have an example of a pytorch compiled package that has an osx arm64 build?

Reply:
There's a bunch of them if you go to the conda-forge GH org and search for repos starting with torch or pytorch (or you can look which feedstocks get touched by the pytorch migration on conda-forge.org/status)

Reply:
I've often seen it in build as well (presumably for anything that actually needs to execute some code from pytorch during installation), but for linking we need the one from host.

The right way would probably be to re-add the build dependency but ensure that whatever feeds the linkage mechanism looks in host first (might need a small patch)

Reply:
The build dependency is already here.

The current hack works, and it's already used by other packages. I understand it's far from ideal, but I don't really see another way at the moment.